tests/e2e/constants.go

```diff
@@ -1,19 +0,0 @@
-package e2e
```

Comment:
These constants were moved to the framework package so they can be shared across parallel and serial tests.